src/markdown_generators.js

```diff
@@ -82,8 +82,8 @@ turndownService.addRule("refshortcode", {
     const ref = turndownService.escape(
       node
         .getAttribute("href")
-        .replace(REFSHORTCODESTART, '{{< ref "')
-        .replace(REFSHORTCODEEND, '" >}}')
+        .replace(REFSHORTCODESTART, '{{% ref "')
+        .replace(REFSHORTCODEEND, '" %}}')
```

Comment:
just curious, why was this change necessary?

Reply:
It wasn't actually necessary per se, but:

> In Hugo 0.55 we changed how the % delimiter works. Shortcodes using the % as the outer-most delimiter will now be fully rendered when sent to the content renderer

I figured we should standardize on using shortcodes this way, even if it doesn't really matter for the `ref` shortcode.  This means that the output of the shortcode will be rendered as markdown fully before the final markdown processing step.